therapi-runtime-javadoc/src/main/java/com/github/therapi/runtimejavadoc/RuntimeJavadoc.java

```diff
@@ -95,4 +123,48 @@ private static String getResourceName(String qualifiedClassName) {
     private static Optional<MethodJavadoc> findMethodJavadoc(List<MethodJavadoc> methodDocs, Method method) {
         return methodDocs.stream().filter(m -> m.matches(method)).findAny();
     }
+
+    /**
+     * Gets the Javadoc of the given field.
+     * <p>
+     * Implementation note: this method first retrieves the Javadoc of the class, and then matches the field name
+     * with the correct documentation. If the client code's purpose is to loop through all fields doc, prefer using
+     * {@link #getJavadoc(Class)} (or one of its overloads), and calling {@link ClassJavadoc#getFields()} on the
+     * returned class doc to retrieve field docs.
```

Comment:
Isn't here and elsewhere a `</p>` missing?

Reply:
It depends on the style guide one follows. I tend to follow the one advising against closing P tags.